products.meta

```diff
@@ -137,7 +138,8 @@
         "chef/infra-proxy-service",
         "chef/config-mgmt-service",
         "chef/data-feed-service",
-        "chef/event-gateway"
+        "chef/event-gateway",
+        "chef/automate-cds"
```

Comment:
maybe we should put this in the dev collection until its ready?

Reply:
@jaym I did not know there was a dev collection. I moved it thank you.